linux/scripts/launchpad.sh

```diff
@@ -25,7 +25,7 @@ if [[ -z "${TIER}" ]]; then
     # In master branch this should be alpha and the debian/watch files should check alpha dir
 #    tier="alpha"
 #    tier="beta"
-    tier="stable"
+    tier=`cat ../TIER.md`
```

Comment:
Don't we want the default tier to be "stable"?

Reply:
The tier should always reflect the source tier, shouldn't it?

Reply:
yeah, if it's coming from `../TIER.md`, maybe the ${TIER} parameter is redundant and can be removed

Reply:
I can do some more redundancy cleanup in a future PR -- once this is working I'd like to get it merged so we can remove the 14.0 alpha block :)